shared_tasks/dynalab/README.md

```diff
@@ -89,16 +89,45 @@ based your model's specific dependencies.
 Once you've implemented the handler you'll need to test it locally.
 
 First install `dynalab` using instructions from their [repo](https://github.com/facebookresearch/dynalab#installation).
+### Project installation and dependencies 
+
+#### 1. Create & activate virtual venv 
+```angular2html
+python3 -m venv venv
+source venv/bin/activate
+```
+#### 2. Install Dynalab
+```angular2html
+git clone https://github.com/facebookresearch/dynalab.git
+cd dynalab
+pip install -e .
+```
+#### 3. Install fairseq & sentencepiece 
+```angular2html
+pip install sentencepiece 
+pip install fairseq
+```
+#### 4. Clone flores repo & put your model files inside the shared_task/dynalab folder
+```angular2html
+git clone https://github.com/facebookresearch/flores
+cd shared_task/dynalab
+```
 
 The simplest test is to run `python handler.py`.
 You'll need to update the `local_test` function to use the task you want.
 Then you can move to running more involved tests using Dynalab.
 
 Afterwards, from this directory run:
-`dynalab-cli init -n <name-of-your-model>`
+```angular2html
+dynalab-cli init -n <name_of_your_model> --model-checkpoint <model.pt_file> -t <flores_task_code> --model-files <your_comma_seperated_model_files>
+
+example:
+
+dynalab-cli init -n asimafrican --model-checkpoint model.pt -t flores_african --model-files "model_generation.json","dict.txt","sentencepiece.bpe.model"
+```
 Note that the model name needs to be lower-kebab-case.
 
-Chose the track you want to apply to: "flores_small1", "flores_small2" or "flores_full".
+Chose the track you want to apply to: "flores_small1", "flores_small2", "flores_full" or "flores_african"
```

Comment:
can you make this specific to `flores_african` ? and list `flores_african` every where needed ?